### PR TITLE
Create a executable utility to call optimizer

### PR DESCRIPTION
**Pull request by justinchuby**
Usage:

```
python optimize.py model.onnx optimized_model.onnx
```

### CODE REVIEW

File: tools/optimize.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    model = onnx.load(path, load_external_data=False)
+    # Hack: Change the working directory to the model directory so the optimizer
+    # can load external data files with relative paths.
+    # TODO: Remove this hack by fixing the optimizer to handle external data files properly.
+    pwd = os.getcwd()
+    model_dir = os.path.dirname(path)
+    os.chdir(model_dir)
+    model = onnxscript.optimizer.optimize(model)
+    model = onnx.inliner.inline_local_functions(model)
```
Comment (titaiwangms) [inline tools/optimize.py:32]: Is this necessary?

**Review by titaiwangms — APPROVED** [review]
Do we plan to use it anywhere?

File: tools/optimize.py (hunk 1 above)
Comment (justinchuby) [reply tools/optimize.py:32]: Since the current optimizer will specialize functions and retain them I wanted to flatten everything with this

**Comment by justinchuby** [comment]
> Do we plan to use it anywhere?

Just a handy script. I am using it to understand the difference between torchscript models and the new onnx models

**Pull request merged**